### ISSUE TITLE
Migration from seafile to docker seafile, can't run seaf fsck

### ISSUE THREAD

**Issue by kopax** [JensErat/docker-seafile#13]
Hi.
I was using seafile in https behind a nginx proxy with fastcgi.
For the moment I have installed your image and run the installation.
I link through docker compose my previous data directory.
I also added a mariadb container and restore my previous database.

I have modified my nginx proxy to redirect to the new docker seafile host.

There are some files I can't watch in web ui, the client is connected.

I am running the check to see if db is corrupted as discrebed in seafile [doc](http://manual.seafile.com/maintain/seafile_fsck.html)

    root@seafile:/opt/seafile/seafile-server-latest# ./seaf-fsck.sh 

    Starting seaf-fsck, please wait ...

    [10/11/15 13:59:07] seaf-fsck.c(155): Current user (0) is not the user for running seafile server (1000). Unable to run fsck.
    seaf-fsck run done

    Done.

**Comment by camory** [comment]
I get the same issue. I have lost some libraries. In the admin the libraries are still there but with the name None. I wanted to check the data integrity but I got the same error message when running seaf-fsck.sh with the root user

**Comment by DorianVasco** [comment]
Old thread, but I found a work around: you have to chown the seafile-data directory to root. Than seaf-fsck will be executable as root, too.

**Comment by raytrax** [comment]
yep old thread, but u must run fsck with the user who owns the installation. You can check it with a "ls -l" and switch to this user "su username" and run fsck ;-)

[Close the thread]
